lib/pure/asyncnet.nim

```diff
@@ -263,7 +263,10 @@ when defineSsl:
       # TODO: Introduce an async template transformation pragma?
 
       # Send any remaining pending SSL data.
-      yield sendPendingSslData(socket, flags)
+      block:
+        let fut = sendPendingSslData(socket, flags)
+        yield fut
+        fut.read() # Check for errors
```

Comment:
Nowadays you can just replace this with `await` afaik

Reply:
I missed that somehow. Fixed.